tests/api_tests/test_questionnaire_response_api.py

```diff
@@ -1707,6 +1707,73 @@ def test_questionnaire_life_functioning_survey(self):
+        self.assertEqual(response_obj.answers[1].valueDate, datetime.date(2022, 11, 30))
+        # Get request from API to assert information is accurate
+        summary = self.send_get("Participant/%s/Summary" % participant_id)
+        self.assertEqual(summary['remoteIdVerificationOrigin'], 'vibrent')
+        self.assertEqual(summary['remoteIdVerificationStatus'], 'TRUE')
+        self.assertEqual(summary['remoteIdVerifiedOn'], '2022-11-30')
+        # Reset Config back to original user client ID
+        user_info['example@example.com']['clientId'] = original_user_client_id
+        config.override_setting(config.USER_INFO, user_info)
+
+    def test_remote_identity_not_verified(self):
+        """ Test to see if a remote ID verification False Response saves successfully """
+        # Set up user config to have client set to vibrent
+        user_info = config.getSettingJson(config.USER_INFO)
+        original_user_client_id = user_info['example@example.com']['clientId']
+        user_info['example@example.com']['clientId'] = 'vibrent'
+        config.override_setting(config.USER_INFO, user_info)
+        # Set up participant, questionnaire, questionnaire response & send POST request to API
+        participant = self.data_generator.create_database_participant(participantOrigin='vibrent')
+        participant_id = f'P{participant.participantId}'
+        authored = datetime.datetime.now()
+        self.send_consent(participant.participantId, authored=authored)
+        # Set up a questionnaire that usually changes participant summary
+        questionnaire_id = self.create_questionnaire("remote_id_verification_questionnaire.json")
+        resource = self._load_response_json("remote_id_verification_questionnaire_false_response.json",
+                                            questionnaire_id,
+                                            participant_id)
+        self._save_codes(resource)
+        response = self.send_post(_questionnaire_response_url(participant_id), resource)
+        # Check the questionnaire response table to make sure data is accurate
+        response_obj = self.session.query(QuestionnaireResponse).filter(
+            QuestionnaireResponse.questionnaireResponseId == response['id']
+        ).one()
+        self.assertEqual(response_obj.answers[0].valueDecimal, 0)
+        self.assertEqual(response_obj.answers[1].valueDate, datetime.date(2022, 11, 30))
+        # Get request from API to assert information is accurate
+        summary = self.send_get("Participant/%s/Summary" % participant_id)
+        self.assertNotIn('remoteIdVerificationOrigin', summary.keys())
+        self.assertNotIn('remoteIdVerifiedOn', summary.keys())
+        self.assertEqual(summary['remoteIdVerificationStatus'], 'UNSET')
+        # Reset Config back to original user client ID
+        user_info['example@example.com']['clientId'] = original_user_client_id
+        config.override_setting(config.USER_INFO, user_info)
+
```

Comment:
Think we discussed moving this to `test_participant_summary_api.py`

Reply:
Yes, Sry, trying to figure out how to unpack and submit the response answers in the participant summary test file using the code answers, date answers, etc. Left these asserts in here for now.